community/releases/roadmap.md

```diff
@@ -14,66 +14,67 @@ please treat this roadmap as aspirational.
-  [Slack](http://join.spinnaker.io).
-
-### GCE
-
-Continue stabilizing the GCE provider in Spinnaker and converging on feature
-parity with the platform.
-
-* Implement integrations for GCE alpha features.
-* Round out integration test coverage.
-
-### Canary
-
-Support deploying the baseline & canary clusters automatically as a part of the
-canary analysis stage.
-
-### CLI (spin)
-
-Reduce friction of automating workflows that programmatically interact with
-Spinnaker's API
-
-* Support authentication.
-* Harden the error handling in the API flows the CLI exercises.
-* Harden the API surface the CLI interacts with.
-* Round out integration test coverage.
-
-### Orca
+# 1.10.0 (Feature-freeze 10-01-18)
+
+## Artifacts
+
+We are continuing to improve artifact support in Spinnaker, along two major
+threads of work:
+Improving the ability to pass build artifacts from CI systems to Spinnaker
+Adding information on artifact provenance to Spinnaker workflows
+
+## Canary
+
+We are working on stages that provision the infrastructure necessary to perform
+a canary (both the baseline and canary clusters) to make configuration easier
+and reduce duplication in pipelines. This is currently targeted only at
+VM-based providers, but Kubernetes support will come in a later release.
+
+## Kubernetes
+
+In order to better surface relevant information in the Spinnaker UI, we are
+spending much of this release improving and reworking the Infrastructure tab
+for Kubernetes resources. The goal is to make the new Kubernetes provider a
+  first-class citizen of the frontend. This includes, but is not limited to:
+
+* Better representation of the `deployment` object, as it is core to much
+  release automation and has so far been confusing to interact with in the
+  current UI.
+* Surfacing of container-level metrics to give insight into the state of an
+  application.
+* Adding a proper YAML-editor to make changes to existing resources and
+  pipelines easier to perform.
+
+## Cloud Foundry
+
+Pivotal has reworked the Cloud Foundry provider implementation. The provider
+uses Frigga to name assets and provides a multi-foundation view of applications
+and clusters based on this naming convention. The provider implementation
+supports declarative and manifest-based configuration in the deployment stage.
+It supports red/black, highlander, and none deployment strategies along with
+rollback, resize, enable/disable server group actions and a terminate instance
+action. The provider also supports promotion of clusters via droplet-based
+deployments.
+
+The 1.10.0 release contains a preview of this Cloud Foundry support. Expect a
+backwards incompatible change to the deploy stage configuration in a future
+release once Spinnaker's expected artifact model improves to the point where
+Cloud Foundry artifacts and manifests can be fully satisfied by expected
+artifacts. The Cloud Foundry provider should be viewed as incubating until that
```

Comment:
"Expect a backward-incompatible change to Deploy stage configuration in a future release, once Spinnaker's expected-artifact model improves to the point that Cloud Foundry artifacts and manifests can be fully satisfied by expected artifacts."